src/lib/util/RichDisplay.js

```diff
@@ -157,7 +157,14 @@ class RichDisplay {
 	 * @private
 	 */
 	_determineEmojis(emojis, stop) {
-		if (this.pages.length > 1 || this.infoPage) emojis.push(this.emojis.first, this.emojis.back, this.emojis.forward, this.emojis.last, this.emojis.jump);
+		if (this.pages.length > 1 || this.infoPage) {
+			for (const prop in this.emojis) {
```

Comment:
Objects do not maintain order. This will lead to the reactions being somewhat randomized.